Improve image display

parth-07 opened the pull request:
Improved image display gui using [blueimp/gallery](https://github.com/blueimp/Gallery)

**Features**
- Swipe events added for changing images 
- More smooth and uniform transitions , uniform as in image size remains uniform
- Slideshow can be enabled using gui controls or space bar
- gui controls for directly skipping more than 1 images

**Screenshots** 
![Screenshot from 2020-12-13 00-42-56](https://user-images.githubusercontent.com/40723498/101992913-882c2c00-3cdc-11eb-8e8c-ea74c7b62677.png)
![Screenshot_20201213-004635_Chrome](https://user-images.githubusercontent.com/40723498/101993081-bfe7a380-3cdd-11eb-9f6e-e01da00d0298.jpg)

Closes #20

manorit2001 commented on src/user/views/displayImage.mustache:
```diff
@@ -61,85 +72,74 @@
+            temp.href = '/public' + file;
+            galleryItems.push(temp);
+        }
+        var gallery = blueimp.Gallery(
```
- Image name is already being shown in the title bar and having image name shown inside the image also feels kinda repetitive
- Same is the case with arrows as the image thumbnails look a lot better and we don't need arrows, the arrows also feels repetitive and unrequired

If there is a `blueimp` settings, then I would propose these changes to be incorporated before merging this branch to master

manorit2001 reviewed (changes requested):
Please look into the changes proposed

parth-07 commented:
Yeah , I agree with all your suggested changes . 
I have removed image name from inside the image as well as next and previous controls inside the image .

Updated screenshots :

![Screenshot from 2020-12-13 13-23-05](https://user-images.githubusercontent.com/40723498/102006331-68c9e900-3d46-11eb-9ef2-679d9695f51e.png)

manorit2001 reviewed (approved):
Looks good
